crates/bevy_sprite/src/bundle.rs

```diff
@@ -38,7 +38,7 @@ impl Default for SpriteBundle {
 /// to as a `TextureAtlas`)
 #[derive(Bundle, Clone, Default)]
 pub struct SpriteSheetBundle {
-    /// The specific sprite from the texture atlas to be drawn
+    /// The specific sprite from the texture atlas to be drawn, defaulting to the first sprite in the texture atlas.
```

Comment:
Maybe index 0 instead of first would be clearer:
```suggestion
    /// The specific sprite from the texture atlas to be drawn, defaulting to the sprite with index 0.
```